README.md

```diff
@@ -190,35 +190,22 @@ let! recursive = GenX.auto<Recursive>
 // (subject to change).
 ```
 
-**Generate any type automatically and override default auto-generators:**
+**Generate any type automatically and override default generators and settings:**
 
 ```f#
-let! myVal = GenX.autoWith<MyType> {GenX.defaults with String = ...; SeqRange = ...}
-```
-
-The current defaults are (see Gen.fs for up-to-date list):
-
-```f#
-let defaults =
-  { Byte = Gen.byte <| Range.exponentialBounded ()
-    Int16 = Gen.int16 <| Range.exponentialBounded ()
-    UInt16 = Gen.uint16 <| Range.exponentialBounded ()
-    Int = Gen.int <| Range.exponentialBounded ()
-    UInt32 = Gen.uint32 <| Range.exponentialBounded ()
-    Int64 = Gen.int64 <| Range.exponentialBounded ()
-    UInt64 = Gen.uint64 <| Range.exponentialBounded ()
-    Double = Gen.double <| Range.exponentialBounded ()
-    Decimal = Gen.double <| Range.exponentialBounded () |> Gen.map decimal
-    Bool = Gen.bool
-    Guid = Gen.guid
-    Char = Gen.latin1
-    String = Gen.string (Range.linear 0 50) Gen.latin1
-    DateTime = Gen.dateTime
-    DateTimeOffset = Gen.dateTime |> Gen.map System.DateTimeOffset
-    Uri = uri  // custom generator in this library
-    SeqRange = Range.exponential 0 50
-    RecursionDepth = 1 }
-```
+let! myVal =
+  {GenX.defaults with 
+     SeqRange = Range.exponential 1 10
+     RecursionDepth = 2}
+  // Will use this generator for all ints
+  |> AutoGenConfig.addGenerator (Gen.int (Range.linear 0 10))
+  // Will use this generator when generating its return type
+  |> AutoGenConfig.addGenerator Gen.myCustomGen
+  // Generate using the config above
+  |> GenX.autoWith<MyType>
+```
+
+If you’re not happy with the auto-gen defaults, you can of course create your own generator that calls `GenX.autoWith` with your chosen config and use that everywhere.
```

Comment:
Could possibly link "defaults" to `https://github.com/hedgehogqa/fsharp-hedgehog-experimental/blob/master/src/Hedgehog.Experimental/Gen.fs#LC361:~:text=let%20defaults%20%3D%20%7B`

The text linking only works for chromium browsers, but I think it's better than nothing.

Reply:
Previously the readme specified the defaults. I removed it because 1) it wasn't really a part of the contract and could change at any time (the readme even said so, and referred to the source for an up-to-date answer), and 2) while I considered linking to the source like you suggest, that link is bound to either become out of date (if linking to a specific commit) or start pointing to the wrong part of the file (if not linking to a specific commit).

Reply:
The link I provided uses "[text fragments](https://web.dev/text-fragments/)" to link directly to `let defaults = {` on master's `Gen.fs`. It's kinda like linking to an anchor tag. This makes it more resilient to line number changes, though it will still break if you rename `defaults`.

I don't feel strongly about this, hence me approving your PR. Just making sure you understand my prior comment.

[Linking it again](https://github.com/hedgehogqa/fsharp-hedgehog-experimental/blob/master/src/Hedgehog.Experimental/Gen.fs#:~:text=let%20defaults%20%3D%20%7B) so it's easier to see in action.

Reply:
Thanks, I see. As you say, requires Chromium (at least it doesn't work in FF). Let's do without the link for now; if anyone asks, I'll consider adding it back.